.github/linters/.jscpd.json

```diff
@@ -0,0 +1,12 @@
+{
+  "threshold": 0,
+  "reporters": [
+    "consoleFull"
+  ],
+  "ignore": [
+    "**/__snapshots__/**",
+    "**/cloud_Azure/terraform/module/examples/multiple_accounts_multiple_resource_group/**",
```

Comment:
why ignore?

Reply:
[Here](https://github.com/kentik/config-snippets-cloud/runs/6905360928?check_suite_focus=true#step:4:24379) and [here](https://github.com/kentik/config-snippets-cloud/runs/6905360928?check_suite_focus=true#step:4:24906) You can find errors about it. 
I wouldn't change demo files at all. 
To fix this linter check in python files would need a small refactor of this scripts, so I decided to leave it as it is.